plio/views.py

```diff
@@ -170,7 +170,7 @@ def list_uuid(self, request):
     @action(
         methods=["get"],
         detail=True,
-        permission_classes=[IsAuthenticated, PlioPermission],
+        permission_classes=[PlioPermission],
```

Comment:
removed so that even unauthenticated users can access embedded plios - didn't make sense to create another endpoint for it.